ui/Screen/Project/Source.svelte

```diff
@@ -36,6 +36,14 @@
 
   const { id, metadata } = getContext("project");
 
+  let headerHeight;
+  onMount(() => {
```

Comment:
This can be done more cleanly with: https://svelte.dev/docs#Block-level_element_bindings.

```diff
diff --git a/ui/Screen/Project/Source.svelte b/ui/Screen/Project/Source.svelte
index e0170b1..c4e163a 100644
--- a/ui/Screen/Project/Source.svelte
+++ b/ui/Screen/Project/Source.svelte
@@ -1,5 +1,5 @@
 <script>
-  import { getContext, onMount } from "svelte";
+  import { getContext } from "svelte";
   import { querystring, push } from "svelte-spa-router";
   import { format } from "timeago.js";

@@ -36,14 +36,9 @@

   const { id, metadata } = getContext("project");

+  let scrollY = 0;
   let headerHeight;
-  onMount(() => {
-    headerHeight = document
-      .getElementById("projectHeader")
-      .getBoundingClientRect().height;
-  });

-  let scrollY = 0;
   let currentPeerId;
   let currentRevision;
   let currentObjectType;
@@ -243,7 +238,7 @@

 <Remote store={projectStore} let:data={project}>
   <div class="header-wrapper">
-    <div id="projectHeader" class="header center-content">
+    <div bind:clientHeight={headerHeight} class="header center-content">
       <h2>{project.metadata.name}</h2>
       <div class="description">
         <p>{project.metadata.description}</p>
@@ -258,7 +253,8 @@
   </div>
   <div class="wrapper">
     <div
-      class="repo-header-wrapper {scrollY > headerHeight ? 'elevation' : ''}">
+      class="repo-header-wrapper"
+      class:elevation={scrollY > headerHeight}>
       <div class="repo-header center-content">
         <!-- Revision selector -->
         <Remote store={revisionsStore} let:data={revisions}>
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/772/commits/385fe5a56d3d4f747334fb1625e2ec37b148d2f9